test/cmdlineTests/yul_string_format_ascii_long/output.json

```diff
@@ -28,7 +28,9 @@ object \"C_10\" {
 
             if iszero(lt(calldatasize(), 4))
             {
-                let selector := shift_right_224_unsigned(calldataload(0))
```

Comment:
Can you check what the gas cost difference here is?

Reply:
I did some measurements last week. See https://github.com/ethereum/solidity/pull/7597#discussion_r439523065

Reply:
@chriseth @axic what should we do here now? should we include commit c8d95d601ee540e5d33e05b7fe718adb44a20dea? or will hera be changed in a way that the old code will just work?